cmd/software/deploy.go

```diff
@@ -51,6 +54,9 @@ func NewDeployCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&saveDeployConfig, "save", "s", false, "Save deployment in config for future deploys")
 	cmd.Flags().BoolVarP(&ignoreCacheDeploy, "no-cache", "", false, "Do not use cache when building container image")
 	cmd.Flags().StringVar(&workspaceID, "workspace-id", "", "workspace assigned to deployment")
+	if !context.IsCloudContext() && houston.VerifyVersionMatch(houstonVersion, houston.VersionRestrictions{GTE: "0.34.0"}) {
+		cmd.Flags().BoolVarP(&isDagOnlyDeploy, "dags", "", false, "Deploy only those dags which are present in the dags folder from working directory")
```

Comment:
Can we keep the flag description similar to what we have in Astro.

CC: @jwitz @lzdanski Could use your input here.

Reply:
changed it!